src/munged/conf.c

```diff
@@ -110,6 +111,7 @@ struct option long_opts[] = {
     { "seed-file",         required_argument, NULL, OPT_SEED_FILE     },
     { "syslog",            no_argument,       NULL, OPT_SYSLOG        },
     { "trusted-group",     required_argument, NULL, OPT_TRUSTED_GROUP },
+    { "listen-backlog",    required_argument, NULL, OPT_LISTEN_BACKLOG},
```

Comment:
Please move "listen-backlog" to appear below "key-file" to keep that section of the struct sorted alphabetically.